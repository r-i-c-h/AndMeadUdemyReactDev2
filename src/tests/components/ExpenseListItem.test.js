import React from 'react';
import { shallow } from 'enzyme';
import FAKE from '../FIXTURES/dummyExpensesDataArray';
import ExpenseListItem from '../../components/ExpenseListItem';

test('should render ExpenseListItem with expense', () => {
  const wrapper = shallow( <ExpenseListItem { ...FAKE[0] } /> );
  expect(wrapper).toMatchSnapshot();
});