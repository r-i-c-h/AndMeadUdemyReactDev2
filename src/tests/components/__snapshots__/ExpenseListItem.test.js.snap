// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem with expense 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h3>
      thing-1
    </h3>
  </Link>
  <p>
     $
    1.11
     - 
    0
     
  </p>
</div>
`;
